images/server/install-packages.sh

```diff
@@ -95,11 +95,11 @@ samba_packages=(\
     ctdb)
 case "${package_selection}-${OS_BASE}" in
     *-fedora|allvfs-*)
-        samba_packages+=(samba-vfs-cephfs samba-vfs-glusterfs)
```

Comment:
I think we had gluster enabled on fedora in case someone in the community wanted to tinker. If you think that's not worthwhile, that's fine with me as I didn't plan on using it.

Reply:
Fine. The retirement plan is for Fedora 42 only. I've decided to keep it for another year(maybe).

Reply:
"TIL" about retiring glusterfs from fedora. I'm OK with removing it I just didn't want to remove it as a side effect of adding something else unrelated. :-)